### ISSUE TITLE
windows

### ISSUE THREAD

**Issue by XAB745220zt** [ruinmessi/RFBNet#94]
Can I use it on Windows

**Comment by masoudpz** [comment]
absolutely yes. it's possible

**Comment by TheLostIn** [comment]
Have you successed?

**Comment by masoudpz** [comment]
yes of course, tomorrow I will fork this repository and will change for using in windows, you can clone my version and use it.
please remember me, tomorrow if you can.
tnx
@TheLostIn

**Comment by zhuixunforever** [comment]
@masoudpz hai, have you successed? where to clone your version?

**Comment by masoudpz** [comment]
https://github.com/masoudpz/RFBNet
check this link, i've changed test.py file and utils/nms_wrapper for running on windows.you can check these two files, load trained model and run the code on windows
I can help you if needed.
@zhuixunforever

**Comment by zhuixunforever** [comment]
I'll try it, thank you very much!

**Comment by zhuixunforever** [comment]
how did you compile ./utils/build.py on windows??  just run ./make.sh???
@masoudpz

**Comment by masoudpz** [comment]
./make.sh is needed for coco dataset, you can use rfbnet by other dataset without ./make.sh

**Comment by zhuixunforever** [comment]
that's all right! thanks

**Comment by huanghekun** [comment]
> ./make.sh is needed for coco dataset, you can use rfbnet by other dataset without ./make.sh

Nice man,could u share me a py about RFB demo.py on windows or linux?I really need it ,thanks for your friendly help.Thank you

**Comment by masoudpz** [comment]
> > ./make.sh is needed for coco dataset, you can use rfbnet by other dataset without ./make.sh
> 
> Nice man,could u share me a py about RFB demo.py on windows or linux?I really need it ,thanks for your friendly help.Thank you

you can use below link, I have changed it to use in windows
https://github.com/masoudpz/RFBNet
